internal/indexer/definitions/darkpeers.yaml

```diff
@@ -60,8 +60,9 @@ irc:
               torrentSize: 4.79 GiB
               baseUrl: https://darkpeers.org/
               torrentId: "1704"
+              freeLeechPercent: "100"
```

Comment:
This and the ones in `vars` on line 74 needs to be `freeleechPercent` instead. Minor difference with the lowercase l but it make a difference.

Reply:
thank you I see you already made the changes for me so resolving this